test/podmontest/deploy/values-powerstore-nfs.yaml

```diff
@@ -0,0 +1,17 @@
+podmonTest:
+  image: "$REGISTRY_HOST:$REGISTRY_PORT//podmontest:v0.0.54"
```

Comment:
Reference to tester image has additional slash

Reply:
We updated this with reference to Isilon changes.
https://github.com/dell/karavi-resiliency/blob/main/test/podmontest/deploy/values-isilon.yaml